scenario.py

```diff
@@ -63,66 +63,82 @@ def __init__(self, params, experiment_path):
         if "samples_split_option" in params:
             self.samples_split_option = params["samples_split_option"]
         else:
-            self.samples_split_option = "random"
+            self.samples_split_option = "random"  # default
 
         # For configuring if the data of the partners are corrupted or not (useful for testing contributivity measures)
         if "corrupted_datasets" in params:
             self.corrupted_datasets = params["corrupted_datasets"]
         else:
-            self.corrupted_datasets = ["not_corrupted"] * self.partners_count
+            self.corrupted_datasets = ["not_corrupted"] * self.partners_count  # default
 
         # ---------------------------------------------------
         #  Configuration of the distributed learning approach
         # ---------------------------------------------------
 
+        # Multi-partner learning approach
+        multi_partner_learning_approaches_list = [
+            "fedavg",
+            "seq",
+            "seqavg",
+        ]
+
+        if "multi_partner_learning_approach" in params:
+            approach = params["multi_partner_learning_approach"]
+            if approach in multi_partner_learning_approaches_list:
+                self.multi_partner_learning_approach = approach
+            else:
+                raise Exception(f"Multi-partner learning approach '{approach}' is not in methods list.")
```

Comment:
is not in **multi_partner_learning_approaches_list** list.
Actually somethinkg like "Multi-partner learning approach '{approach}' is not a valid approach" might be clearer

Reply:
OK

Reply:
Done